tap_linkedin_ads/streams.py

```diff
@@ -129,7 +147,10 @@ def merge_responses(data):
         # Loop through each record of the page
         for element in page:
             temp_start = element['dateRange']['start']
-            temp_pivotValue = element['pivotValue']
+            temp_pivotValue = element['pivotValues'][0]
+            # adding pivot and pivot_value to make it compatible with the previous tap version 2.2.0
+            element['pivot'] = pivot
+            element["pivot_value"] = temp_pivotValue
```

Comment:
Making tap compatible with the previous version 2.2.0 is really needed? Because the previous tap version is already deprecated.

Reply:
removed 2.2.0 from comments